docs/user-guide/getting-started-with-docker-and-lets-encrypt.md

```diff
@@ -0,0 +1,181 @@
+# Docker & Traefik
+In this use case, we want to use Traefik as a *layer-7* load balancer with SSL termination for a set of microservices used to run a webapplication. We also want to automatically *discover any services* on the Docker host and let Traefik reconfigure itself automatically when containers get created (or shut down) so HTTP traffic can be routed accordingly. In addition, we want to use Let's Encrypt to automatically generate and renew SSL certificates per hostname.
+
+## Setting up
+In order for this to work, you'll need a server with a public IP address, with Docker installed on it. In this example, we're using the fictitious domain *my-awesome-app.org". In real-life, you'll want to use your own domain and have the DNS configured accordingly so the hostname records you'll want to use point to the aforementioned public IP address.
```

Comment:
The character **"** has to be replaced by a character _*_ to terminate the *italic* section.